flink-runtime/src/test/java/org/apache/flink/runtime/jobmaster/slotpool/SlotPoolBatchSlotRequestTest.java

```diff
@@ -233,16 +239,23 @@ public void testPendingBatchSlotRequestTimeoutAfterSlotRelease() throws Exceptio
 		final ManualClock clock = new ManualClock();
 		final Time batchSlotTimeout = Time.milliseconds(1000L);
 
-		try (final TestingSlotPoolImpl slotPool = new SlotPoolBuilder(directMainThreadExecutor)
-				.setClock(clock)
-				.setBatchSlotTimeout(batchSlotTimeout)
-				.build()) {
+		try (final TestingSlotPoolImpl slotPool = createAndSetUpSlotPool(
+				directMainThreadExecutor,
+				null,
+				batchSlotTimeout,
+				clock)) {
+
 			final ResourceID taskManagerResourceId = SlotPoolUtils.offerSlots(slotPool, directMainThreadExecutor, Collections.singletonList(resourceProfile));
-			final CompletableFuture<PhysicalSlot> firstSlotFuture = SlotPoolUtils.requestNewAllocatedBatchSlot(slotPool, directMainThreadExecutor, resourceProfile);
```

Comment:
This change is unrelated to "Refactor tests of SlotPoolImpl with SlotPoolUtils and SlotPoolBuilder".

Reply:
Removed.